### ISSUE TITLE
Metrics table aggregator script

### ISSUE THREAD

**Issue by nosmokingsurfer** [nosmokingsurfer/icra_2026_paper#3]
For the paper we need  a table with evaluation results of different combinations of algorithms and split results by different datasets.
Latex code of this table must be automatically produced by script. Thus if some experiment was updated the table can be quickly updated and included into the paper automatically.

Python script:
- script goes through `out/` directory and reads `results.pkl` files and maybe individual trajectories pkl files
- aggregates RMSE, ATE, RTE errors:
- [ ] for each dataset: splines, ronin
- [ ] training method: window-based (subseq_len==1) and FGO-loss (subseq_len>1). each subseq_len value grouped individually
- [ ] w/ and w/o diffusion denoising

Tricks:
- good approach is to parse all required experiment artifacts from all folders and put everything into pandas DataFrame  and keep "plane" structure where one row  in dataframe is one individual validation trajectory (need also have its all variants from different epochs in other rows too) with all attributes like metrics, epoch number, parameters of augmentations, learning rate, batch size, checkpoint version and so on.
- save this plane table as a file (for example csv). later all analysis and visualizations can be obtained from this "condensed" single-table file, it can be transferred somewhere else and etc.
- for box-plots, contour plots plotly express can be used - it natively works with pandas
